packages/web-components/src/components/ic-stepper/ic-stepper.tsx

```diff
@@ -231,12 +239,26 @@ export class Stepper {
         }
 
         if (this.hideStepInfo && stepTitleArea !== null) {
-          stepTitleArea.classList.add("visually-hidden");
+          stepTitleArea.classList.add(this.visuallyHidden);
         }
       }
     });
   };
 
+  private setHideStepInfo = (): void => {
+    this.steps.forEach((step) => {
+      const stepTitleArea = step.shadowRoot.querySelector(
+        ".step > .step-title-area"
+      );
+
+      if (this.hideStepInfo && stepTitleArea !== null) {
+        stepTitleArea.classList?.add(this.visuallyHidden);
+      } else {
```

Comment:
this seems to be causing an error in the tests as `hideStepInfo` is `true`, but `stepTitleArea` is `null`